src/licensedcode/data/rules/false-positive_ansible_2.RULE

```diff
@@ -5,4 +5,4 @@
   # - GPLv3
   # - Apache
   # - CC-BY
-license: {{ license }}
+license: \{\{ license \}\}
```

Comment:
In this case the `{{ license }}` is not meant to be a key phrase. As a workaround I've "escaped" the `{{` with a `\`, this as not explicit meaning in the parsing code, it simply trips up the regular expression used to find the tags.

Reply:
Okay, removed them!